3rd-party/libssh/CMakeLists.txt

```diff
@@ -59,8 +59,8 @@ file(STRINGS libssh/CMakeLists.txt libssh_VERSION REGEX "^project\\(libssh")
 file(STRINGS libssh/CMakeLists.txt LIBRARY_VERSION REGEX "^set\\(LIBRARY_VERSION")
 file(STRINGS libssh/CMakeLists.txt LIBRARY_SOVERSION REGEX "^set\\(LIBRARY_SOVERSION")
 
-string(REGEX MATCH "^project\\(libssh VERSION ([0-9]+)\\.([0-9]+)\\.([0-9]+) LANGUAGES C\\)$"
-       libssh_VERSION "${libssh_VERSION}")
+string(REGEX MATCH "^project\\(libssh VERSION ([0-9]+)\\.([0-9]+)\\.([0-9]+) LANGUAGES C CXX\\)$"
+  libssh_VERSION "${libssh_VERSION}")
```

Comment:
This change is due to the regex match failure stems from the change in the internal CMakeLists.txt change of libssh, the line 
`project(libssh VERSION 0.11.1 LANGUAGES C CXX)` is appended with CXX.